client/console/src/test/java/org/apache/syncope/client/console/AbstractTest.java

```diff
@@ -65,6 +89,39 @@ public static void loadProps() throws IOException {
         }
     }
 
+    @Test
```

Comment:
Why this test method is duplicate here from https://github.com/apache/syncope/blob/2_1_X/client/console/src/test/java/org/apache/syncope/client/console/SyncopeConsoleApplicationTest.java#L50-L68 ?